src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -6,6 +6,10 @@ body {
     padding: 60px 0 30px;
 }
 
+code {
```

Comment:
A quick search in the code base tells me that `<code>` is actually used in other places. Unless you want to change all appearances regarding `<code>`, it is better to give a proper class name for the HTML tag.

A technique:
```
.technical-information code {
    color: black;
}
```
In doing this, all `<code>` tags under class `technical-information` will be applied with the style.

Reply:
@joanneong check the other places too. Most likely we can use the same style in all places.

Reply:
I've checked, and the other places are actually mostly in header comments. The one other place where there is a visible UI change would be for contribution questions:

The original:

![image](https://user-images.githubusercontent.com/24493463/36088652-0fe79dc2-1013-11e8-834e-6291563a95b9.png)

After changing the style:

![image](https://user-images.githubusercontent.com/24493463/36088660-15506bc2-1013-11e8-90e9-2aad2cff54a5.png)

which I am okay with. What do @damithc @xpdavid think?

Reply:
I am OK with it.

Another small suggestion, the background of the `<code>` element is still a little bit red, which looks a little bit weird with the black font. See if you can change the background to `grey` or something.

Reply:
A simple page inspection gives this:

![image](https://user-images.githubusercontent.com/24493463/36113799-fe043754-1068-11e8-8df6-dda019bb2685.png)

Googling it, I see that you're right: 

![image](https://user-images.githubusercontent.com/24493463/36113849-2d29d17e-1069-11e8-8e91-f20b68199a19.png)

though I didn't notice that at first because I am mildly colour-blind. I will update this accordingly soon!